doc/ck_pr_fence_unlock

```diff
@@ -36,78 +36,37 @@ Concurrency Kit (libck, \-lck)
 .Ft void
 .Fn ck_pr_fence_unlock void
 .Sh DESCRIPTION
-This function enforces the ordering of any memory operations preceding and related to
-the release of a lock relative to the invocation of the function.
-Any realted operations (or operations in the critical section) that were performed prior to the invocation of
+This function serves as an optimization surrounding atomic unlock operations.
```

Comment:
Same thing here, let's drop this sentence. Let's apply same spirit of changes as the corresponding lock fence.